atomic_reactor/plugins/build_source_container.py

```diff
@@ -29,8 +30,11 @@ def run(self):
         Returns:
             BuildResult
         """
-        source_data_dir = tempfile.mkdtemp()  # TODO: from pre_* plugin
-        # TODO fail when source dir is empty
+        fetch_sources_result = self.workflow.prebuild_results.get(PLUGIN_FETCH_SOURCES_KEY, {})
+        source_data_dir = fetch_sources_result.get('image_sources_dir')
```

Comment:
`None` should be handled here, otherwise it will hardfail in next step

```python
>>> os.path.isdir(None)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```